ts/components/session/SessionChannelSettings.tsx

```diff
@@ -50,15 +50,19 @@ export class SessionChannelSettings extends React.Component<Props, any> {
   }
 
   public componentDidUpdate() {
-    this.getMediaGalleryProps()
-      .then(({ documents, media, onItemClick }) => {
-        this.setState({
-          documents,
-          media,
-          onItemClick,
-        });
-      })
-      .ignore();
+    const mediaScanInterval = 1000;
```

Comment:
Is this meant to be a regular interval (as the name suggests) or a one off?

Reply:
This is an interval, as the componentWillUpdate() runs in a loop according to this interval.